src/app/controllers/SurveyController.js

```diff
@@ -62,7 +108,7 @@ class SurveyController {
     createSurveyResponse(survey_id, survey_response) {
         return new Promise((resolve, reject) => {
             let mutationString = `
-                mutation createSurveyResponse ($survey_id: String!, $survey_response: CreateSurveyResponseInput!) {
+                mutation createSurveyResponseMutation ($survey_id: String!, $survey_response: CreateSurveyResponseInput!) {
```

Comment:
I just realized `createSurveyResponseMutation` is not needed here, simply remove, and make it `mutation ($survey_id: ..........)`